source/aggregation-pipeline-builder.txt

```diff
@@ -183,6 +183,39 @@ Open a Saved Pipeline
    To learn more, see 
    :ref:`viewing saved aggregations<saved-query-view>` for details.
 
+.. _set-maxtime-ms-agg:
+
+Set ``maxTimeMS`` for Aggregations Queries
+------------------------------------------
+
+Use the ``maxTimeMS`` parameter on the aggregation tab to put an 
+upper time limit in milliseconds on the aggregation pipelines that 
+run in Compass.
+
+To enable ``maxTimeMS`` on the aggregation tab:
+
+.. procedure::
+   :style: normal
+
+   .. step:: In the aggregation pipeline pane, click the 
+      :guilabel:`More Options`.
+
+      .. figure:: /images/compass/agg-builder-click-more-options.png
+         :alt: Toggle more options
+
+   .. step:: Enter a value next to :guilabel:`Max Time MS` to set a 
+      maximum amount of time in milliseconds that an aggregation 
+      pipeline can run. For a ``5`` second limit enter ``5000``.
+
+      .. figure:: /images/compass/agg-builder-enter-maxtime-ms.png
+         :alt: Enter a maximum timeout value 
+
+   .. step:: Any aggregation pipeline that goes over the time limit 
```

Comment:
[suggestion/nit] It seems like "operation" might be better suited here since the timeout is dependent on the operation that runs, rather than the pipeline itself?
```suggestion
   .. step:: Any aggregation operation that goes over the time limit 
```

Reply:
Its funny, I originally had this as operation and think I changed it right at the end. Updated!

Reply:
Haha that's definitely happened to me before, too. Terminology always gets mixed up in my head 😵‍💫